src/core/server/saved_objects/export/point_in_time_finder.ts

```diff
@@ -170,8 +170,8 @@ export class PointInTimeFinder {
         sortOrder: 'desc',
         // Bump keep_alive by 2m on every new request to allow for the ES client
         // to make multiple retries in the event of a network failure.
-        ...(id ? { pit: { id, keepAlive: '2m' } } : {}),
-        ...(searchAfter ? { searchAfter } : {}),
+        pit: id ? { id, keepAlive: '2m' } : undefined,
+        searchAfter,
```

Comment:
@lukeelmers Any reason for the way we were doing this before where we were avoiding specifying these keys at all when `id` or `searchAfter` was undefined? Curious if there's a possibility of breaking anything when specifying `pit: undefined`. Tests are all passing so I hope not 🙃

Reply:
> Any reason for the way we were doing this before

@joshdover No technical reason for doing it; I had only done it this way to keep the returned query DSL as "clean" as possible. The changes here should be fine.

Reply:
Great!